### PR TITLE
Fix github links

### LINKED ISSUE #131: 링크 수정
https://github.com/nodejs/io.js 가 https://github.com/nodejs/node 로 변경되었습니다.

그랩해보니 이전 링크가 꽤나 나오던데 일괄 수정하는게 좋을것 같습니다.

```
articles/_posts/2015-05-15-Growing-Up.md:94:그래서 우리는 조직되어 있고 우리가 참여하길 바라는 재단이 필요합니다. 개인적인 생각으로 io.js를 위한 이상적인 구조로 되어 있으면 좋겠습니다. 추가로 여전히 양분된 node.js와 io.js 커뮤니티의 분열과 혼란이 종식되기를 바랍니다. [만약 선택할 수 있다면](https://github.com/nodejs/io.js/issues/1664), 우리는 이 모든 것을 얻고 싶고 관리 방법, 배포 과정, 그리고 실행 그룹을 온전히 지켜내고 싶습니다.
weekly/_posts/2015-05-22-weekly.md:44:이번 주 [v1.8.2](https://iojs.org/dist/v1.8.2/), [v2.0.2](https://iojs.org/dist/v2.0.2/), [v2.1.0](https://iojs.org/dist/v2.1.0/) 3개의 io.js를 릴리스했습니다. 전체 변경사항은 [GitHub](https://github.com/nodejs/io.js/blob/master/CHANGELOG.md)에서 확인할 수 있습니다.
weekly/_posts/2015-05-22-weekly.md:50:* **암호화**: TLS의 메모리 사용을 상당히 줄임 (Fedor Indutny & Сковорода Никита Андреевич) [#1529](https://github.com/nodejs/io.js/pull/1529)
weekly/_posts/2015-05-22-weekly.md:56:* **win,node-gyp**: 기본값으로 윈도우 애드온의 딜레이-로드 훅이 이제 제대로 활성화 됩니다. 2.0.0에서는 실수로 비활성화되었습니다. (Bert Belder)[#1433](https://github.com/nodejs/io.js/pull/1433)
weekly/_posts/2015-05-22-weekly.md:57:* **os**: 임시 디렉터리가 '/'에 있을 때의 이슈를 고치려고 tmpdir()의 마지막 슬래시(/) 제거를 개선했습니다. 또한 어떤 슬래시가 운영체제에
서 사용되는지 고려합니다. (cjihrig) [#1673](https://github.com/nodejs/io.js/pull/1673)
weekly/_posts/2015-05-22-weekly.md:58:* **tls**: gcm과 aes128을 사용하려고 기본 암호화를 업데이트 했습니다. (Mike MacCana) [#1660](https://github.com/nodejs/io.js/pull/1660)
weekly/_posts/2015-05-22-weekly.md:59:* **build**: 이전의 보안 이슈를 해결한 이후 v8 팀이 제안한 대로 v8 스냅샷을 기본적으로 다시 사용할 수 있게 했습니다. 이로써 구동과 vm 컨
텍스트 생성 모두에 성능 향상이 약간 있을 것입니다. (Trevor Norris) [#1663](https://github.com/nodejs/io.js/pull/1663)
weekly/_posts/2015-05-22-weekly.md:60:* **src**: `--require`옵션을 사용하기 전에 다른 플래그를 사용하면 프리로드 옵션이 동작하지 않던 문제를 수정했습니다. (Yosuke Furukawa) [#1694](https://github.com/nodejs/io.js/pull/1694)
weekly/_posts/2015-05-22-weekly.md:61:* **dgram**: `send()`콜백이 비동기로 동작하지 않던 문제를 수정했습니다. (Yosuke Furukawa) [#1313](https://github.com/nodejs/io.js/pull/1313)
weekly/_posts/2015-05-22-weekly.md:62:* **readline**: emitKey는 버퍼링 데이터를 충분히 파싱할 때까지 버퍼링 데이터를 가지고 있습니다. 이 수정으로 파싱 스플릿 이스케이프에 관한 문제도 해결됩니다. (Alex Kocharin) [#1601](https://github.com/nodejs/io.js/pull/1601)
weekly/_posts/2015-05-22-weekly.md:63:* **cluster**: cluster.worker에게 '디스커넥트'를 올바르게 보냅니다. (Oleg Elifantiev) [#1386](https://github.com/nodejs/io.js/pull/1386)
weekly/_posts/2015-05-22-weekly.md:64:* **events**: 잡히지 않은 에러(uncaught errors)도 이제 약간의 컨텍스트를 제공합니다. (Evan Lucas) [#1654](https://github.com/nodejs/io.js/pull/1654)
weekly/_posts/2015-05-22-weekly.md:67:* **crypto**: 디피-헬만 키 교환(DHE) 매개변수(‘dhparams’)는 1024비트 이거나 그 이상이어야 합니다. 그 외에는 에러를 던지게 됩니다. 2048비
트 이하로 사용할 경우 경고 메시지를 콘솔에 출력하게 됩니다. 이 보안 문제에 대해서는 [https://weakdh.org/](https://weakdh.org/)에서 자세히 보실 수 있습니다. (Shigeki Ohtsu) [#1739](https://github.com/nodejs/io.js/pull/1739)
weekly/_posts/2015-05-22-weekly.md:68:* **node**: 새로운 `--trace-sync-io` 커맨드 라인 플래그는 경고를 출력하고 동기 API가 사용될 때 스택을 추적합니다. 애플리케이션을 느리게
만드는 동기 호출을 추적할 때 사용할 수 있습니다. (Trevor Norris) [#1707](https://github.com/nodejs/io.js/pull/1707)
weekly/_posts/2015-05-22-weekly.md:69:* **node**: 메소드 체이닝을 위해  ‘net’, ‘dgram’, ‘http’, ‘https’, ‘tls’에서 사용한 setTimeout(), setKeepAlive(), setNoDelay(), ref(), unref() 메소드는 undefined 대신 현재의 인스턴스를 반환합니다. (Roman Reiss & Evan Lucas) [#1699](https://github.com/nodejs/io.js/pull/1699) [#1768](https://github.com/nodejs/io.js/pull/1768) [#1779](https://github.com/nodejs/io.js/pull/1779)
weekly/_posts/2015-05-22-weekly.md:71:* **util**: console.log()에서 사용되는 util.format()에 한 개의 문자열 인자를 넘기는 일반적인 경우를 위한 속도 향상(35% 정도)이 있었습니다. (Сковорода Никита Андреевич) [#1749](https://github.com/nodejs/io.js/pull/1749)
weekly/_posts/2015-05-22-weekly.md:76:See https://github.com/nodejs/io.js/labels/confirmed-bug for complete and current list of known issues.
weekly/_posts/2015-05-22-weekly.md:84:현재 알려진 이슈의 전체 목록은 [https://github.com/iojs/io.js/labels/confirmed-bug](https://github.com/nodejs/io.js/labels/confirmed-bug)에서 보실 수 있습니다.
weekly/_posts/2015-05-22-weekly.md:102:* 디피-헬만 키 교환(DHE)에서 [Logjam](https://weakdh.org/) 공격의 취약점이 밝혀졌습니다. io.js는 [v2.1.0](https://github.com/nodejs/io.js/blob/master/CHANGELOG.md#2015-05-24-version-210-rvagg)에서 [취약점을 수정](https://github.com/nodejs/io.js/pull/1739)했습니다.
weekly/_posts/2015-05-29-weekly.md:13:This week we had two io.js releases [v2.2.0](https://iojs.org/dist/v2.2.0/) and [v2.2.1](https://iojs.org/dist/v2.2.1/), complete changelog can be found [on GitHub](https://github.com/nodejs/io.js/blob/master/CHANGELOG.md).
weekly/_posts/2015-05-29-weekly.md:18:이번 주에는 io.js의 [v2.2.0](https://iojs.org/dist/v2.2.0/), [v2.2.1](https://iojs.org/dist/v2.2.1/) 버전을 릴리스했습니다. 전체 변경사항은 [GitHub](https://github.com/nodejs/io.js/blob/master/CHANGELOG.md)에서 확인할 수 있습니다.
```

### PR DESCRIPTION
**Pull request by marocchino**
close #131, #153

``` shell
sed -i '' -e "s/github.com\/iojs\/io.js/github.com\/nodejs\/node/g" */_posts/*
sed -i '' -e "s/github.com\/iojs/github.com\/nodejs/g" */_posts/*
sed -i '' -e "s/github.com\/iojs/github.com\/nodejs/g" *.md
sed -i '' -e "s/iojs-ko/nodejs-ko/g" */_posts/*.md
```

### CODE REVIEW

**Comment by marocchino** [comment]
@yous 확인부탁드립니다.

File: weekly/_posts/2015-02-13-weekly.md [hunk 1]
```diff
@@ -88,35 +88,35 @@ Language Communities
...
+- [iojs-fi](https://github.com/nodejs/iojs-fi) 핀란드어 커뮤니티
+- [iojs-fr](https://github.com/nodejs/iojs-fr) 프랑스어 커뮤니티
+- [iojs-he](https://github.com/nodejs/iojs-he) 히브리어 커뮤니티
+- [iojs-hi](https://github.com/nodejs/iojs-hi) 힌두어 커뮤니티
+- [iojs-hu](https://github.com/nodejs/iojs-hu) 헝가리어 커뮤니티
+- [iojs-id](https://github.com/nodejs/iojs-id) 인도네시아어 커뮤니티
+- [iojs-it](https://github.com/nodejs/iojs-it) 이탈리아어 커뮤니티
+- [iojs-ja](https://github.com/nodejs/iojs-ja) 일본어 커뮤니티
+- [iojs-ka](https://github.com/nodejs/iojs-ka) 조지아어 커뮤니티
+- [iojs-ko](https://github.com/nodejs/nodejs-ko) 한국어 커뮤니티
```
Comment (marocchino) [inline weekly/_posts/2015-02-13-weekly.md:108]: 이부분은 수동으로 돌려놨습니다.

File: weekly/_posts/2015-02-13-weekly.md [hunk 2]
```diff
@@ -88,35 +88,35 @@ Language Communities
...
-- [iojs-ru](https://github.com/iojs/iojs-ru) 러시아어 커뮤니티
-- [iojs-sv](https://github.com/iojs/iojs-sv) 스웨덴어 커뮤니티
-- [iojs-tr](https://github.com/iojs/iojs-tr) 터키어 커뮤니티
-- [iojs-tw](https://github.com/iojs/iojs-tw) 대만어 커뮤니티
-- [iojs-uk](https://github.com/iojs/iojs-uk) 우크라이나어 커뮤니티
+- [iojs-bn](https://github.com/nodejs/iojs-bn) 벵골어 커뮤니티
+- [iojs-cn](https://github.com/nodejs/iojs-cn) 중국어 커뮤니티
+- [iojs-cs](https://github.com/nodejs/iojs-cs) 체코어 커뮤니티
+- [iojs-da](https://github.com/nodejs/iojs-da) 덴마크어 커뮤니티
+- [iojs-de](https://github.com/nodejs/iojs-de) 독일어 커뮤니티
```
Comment (yous) [inline weekly/_posts/2015-02-13-weekly.md]: https://github.com/nodejs/nodejs-de

File: weekly/_posts/2015-02-13-weekly.md [hunk 3]
```diff
@@ -88,35 +88,35 @@ Language Communities
...
+- [iojs-el](https://github.com/nodejs/iojs-el) 그리스어 커뮤니티
+- [iojs-es](https://github.com/nodejs/iojs-es) 스페인어 커뮤니티
+- [iojs-fa](https://github.com/nodejs/iojs-fa) 페르시아어 커뮤니티
```
Comment (yous) [inline weekly/_posts/2015-02-13-weekly.md]: https://github.com/nodejs/nodejs-fa

File: weekly/_posts/2015-02-13-weekly.md [hunk 4]
```diff
@@ -88,35 +88,35 @@ Language Communities
...
+- [iojs-fi](https://github.com/nodejs/iojs-fi) 핀란드어 커뮤니티
```
Comment (yous) [inline weekly/_posts/2015-02-13-weekly.md]: https://github.com/nodejs/nodejs-fi

File: weekly/_posts/2015-02-13-weekly.md [hunk 5]
```diff
@@ -88,35 +88,35 @@ Language Communities
...
+- [iojs-fr](https://github.com/nodejs/iojs-fr) 프랑스어 커뮤니티
```
Comment (yous) [inline weekly/_posts/2015-02-13-weekly.md]: https://github.com/nodejs/nodejs-fr

File: weekly/_posts/2015-02-13-weekly.md [hunk 6]
```diff
@@ -88,35 +88,35 @@ Language Communities
...
+- [iojs-he](https://github.com/nodejs/iojs-he) 히브리어 커뮤니티
+- [iojs-hi](https://github.com/nodejs/iojs-hi) 힌두어 커뮤니티
+- [iojs-hu](https://github.com/nodejs/iojs-hu) 헝가리어 커뮤니티
+- [iojs-id](https://github.com/nodejs/iojs-id) 인도네시아어 커뮤니티
```
Comment (yous) [inline weekly/_posts/2015-02-13-weekly.md]: https://github.com/nodejs/nodejs-id

File: weekly/_posts/2015-02-13-weekly.md [hunk 7]
```diff
@@ -88,35 +88,35 @@ Language Communities
...
+- [iojs-it](https://github.com/nodejs/iojs-it) 이탈리아어 커뮤니티
+- [iojs-ja](https://github.com/nodejs/iojs-ja) 일본어 커뮤니티
```
Comment (yous) [inline weekly/_posts/2015-02-13-weekly.md]: https://github.com/nodejs/nodejs-ja

File: weekly/_posts/2015-02-13-weekly.md [hunk 8]
```diff
@@ -88,35 +88,35 @@ Language Communities
...
+- [iojs-ka](https://github.com/nodejs/iojs-ka) 조지아어 커뮤니티
```
Comment (yous) [inline weekly/_posts/2015-02-13-weekly.md]: https://github.com/nodejs/nodejs-ka

File: weekly/_posts/2015-02-13-weekly.md [hunk 9]
```diff
@@ -88,35 +88,35 @@ Language Communities
...
+- [iojs-ko](https://github.com/nodejs/nodejs-ko) 한국어 커뮤니티
+- [iojs-mk](https://github.com/nodejs/iojs-mk) 마케도니아어 커뮤니티
```
Comment (yous) [inline weekly/_posts/2015-02-13-weekly.md]: https://github.com/nodejs/nodejs-mk

File: weekly/_posts/2015-02-13-weekly.md [hunk 10]
```diff
@@ -88,35 +88,35 @@ Language Communities
...
+- [iojs-nl](https://github.com/nodejs/iojs-nl) 네렐란드어 커뮤니티
```
Comment (yous) [inline weekly/_posts/2015-02-13-weekly.md]: https://github.com/nodejs/nodejs-nl
네렐란드어 → 네덜란드어

File: weekly/_posts/2015-02-13-weekly.md [hunk 11]
```diff
@@ -88,35 +88,35 @@ Language Communities
...
+- [iojs-no](https://github.com/nodejs/iojs-no) 노르웨이어 커뮤니티
```
Comment (yous) [inline weekly/_posts/2015-02-13-weekly.md]: https://github.com/nodejs/nodejs-no

File: weekly/_posts/2015-02-13-weekly.md [hunk 12]
```diff
@@ -88,35 +88,35 @@ Language Communities
...
+- [iojs-pl](https://github.com/nodejs/iojs-pl) 폴란드어 커뮤니티
```
Comment (yous) [inline weekly/_posts/2015-02-13-weekly.md]: https://github.com/nodejs/nodejs-pl

File: weekly/_posts/2015-02-13-weekly.md [hunk 13]
```diff
@@ -88,35 +88,35 @@ Language Communities
...
+- [iojs-pt](https://github.com/nodejs/iojs-pt) 포르투갈어 커뮤니티
+- [iojs-ro](https://github.com/nodejs/iojs-ro) 루마니아어 커뮤니티
```
Comment (yous) [inline weekly/_posts/2015-02-13-weekly.md]: https://github.com/nodejs/nodejs-ro

File: weekly/_posts/2015-02-13-weekly.md [hunk 14]
```diff
@@ -88,35 +88,35 @@ Language Communities
...
+- [iojs-ru](https://github.com/nodejs/iojs-ru) 러시아어 커뮤니티
+- [iojs-sv](https://github.com/nodejs/iojs-sv) 스웨덴어 커뮤니티
+- [iojs-tr](https://github.com/nodejs/iojs-tr) 터키어 커뮤니티
```
Comment (yous) [inline weekly/_posts/2015-02-13-weekly.md]: https://github.com/nodejs/nodejs-tr

File: weekly/_posts/2015-02-13-weekly.md [hunk 15]
```diff
@@ -88,35 +88,35 @@ Language Communities
...
+- [iojs-tw](https://github.com/nodejs/iojs-tw) 대만어 커뮤니티
+- [iojs-uk](https://github.com/nodejs/iojs-uk) 우크라이나어 커뮤니티
```
Comment (yous) [inline weekly/_posts/2015-02-13-weekly.md]: https://github.com/nodejs/nodejs-uk

File: weekly/_posts/2015-04-17-weekly.md [hunk 16]
```diff
@@ -16,56 +16,56 @@ io.js 1.7 릴리스, 커뮤니티 업데이트와 이벤트 등
...
+* ARMv8 64비트 (AARCH64)를 이제 제대로 지원합니다. libuv에서 `epoll_wait()`의 존재 여부를 오판하던 문제에 대한 수정사항도 포함되었습니다. (Ben Noordhuis) [#1365](https://github.com/nodejs/node/pull/1365). ARMv6: [#1376](https://github.com/nodejs/node/issues/1376)(라스베리 파이 포함)에서 `Math.exp()`를 사용할 때 문제가 발생한다는 보고가 있습니다. V8의 "fast math" 기능을 사용할 때 ARMv6의 잘못된 코드 생성기가 범인이었습니다.  이 문제를 피하기 위해 ARMv6에서는 `—nofast_math`를 기본값으로 사용하도록 했습니다. `—fast_math`를 사용하면 fast math 기능을 사용할 수 있습니다 . (Ben Noordhuis) [#1398](https://github.com/nodejs/node/pull/1398). 테스트: ARMv6, ARMv7과 같은 느린 플랫폼에서 타임아웃 시간을 조정했습니다. (Roman Reiss) [#1366](https://github.com/nodejs/node/pull/1366).
```
Comment (yous) [inline weekly/_posts/2015-04-17-weekly.md]: 사용할 수 있습니다 . → 사용할 수 있습니다.

**Comment by yous** [comment]
다 봤습니다.

**Comment by marocchino** [comment]
반영했습니다 \'ㅁ'/

**Pull request merged**